packages/api/tsconfig.json

```diff
@@ -5,7 +5,7 @@
     "allowJs": true,
     "checkJs": true,
     "target": "esnext",
-    "lib": ["ESNext"],
+    "lib": ["ESNext", "DOM.Iterable"],
```

Comment:
Why please?

Reply:
failed one or more checks with type errors complaining about this I believe:

https://github.com/nftstorage/nft.storage/blob/cb5171ffe480a03f55ebbcde05a5eb762f5721e9/packages/api/src/routes/nfts-store.js#L31